ldk/javascript/src/whisper/types.ts

```diff
@@ -239,6 +239,7 @@ interface SelectComponent<T extends WhisperComponentType> extends WhisperCompone
 export type Autocomplete = SelectComponent<WhisperComponentType.Autocomplete> & {
   label?: string;
   loading?: boolean;
+  multiple?: boolean;
   onChange?: WhisperHandlerWithParam<string>;
   onSelect: WhisperHandlerWithParam<string>;
   options?: AutocompleteOption[];
```

Comment:
Is it possible to add comments to the properties of the Autocomplete so it would be more descriptive for the loop authors?